src/opnsense/service/templates/OPNsense/Unbound/core/dnsbl_module.py

```diff
@@ -11,8 +50,19 @@ def __init__(self, env):
         self.dnsbl_mtime_cache = 0
         self.dnsbl_update_time = 0
         self.dnsbl_available = False
+        self.log_update_time = time.time()
+        self.dnssec_enabled = 'validator' in self.env.cfg.module_conf
+        self.stats_enabled = os.path.exists('/data/stats')
```

Comment:
ah here we are, I see :)

Reply:
leave it as a file check in start.sh then? seems like more code than necessary if we still need the file check anyway

Reply:
yep, nothing to do here